### ISSUE TITLE
Looks like LinkedIn blocked the program

### ISSUE THREAD

**Issue by williamfgm96** [mclmza/linkedin-learning-downloader#48]
I've been trying to download courses but I haven't.
I tried changing the IP, changing the account but it stills imposible to me download any course.

**Comment by mclmza** [comment]
Just tried and is working for me.

**Comment by duracotton** [comment]
My version is also working, please check your account (can you still watch all chapters of a course)?

**Comment by williamfgm96** [comment]
Yes, I have all access to the courses, and I also tried with another account, another IP but everything just got me the same result

**Comment by mclmza** [comment]
Still having problems?

**Comment by williamfgm96** [comment]
@mclmza Yes 😞
I'm not able to download courses and idk why, I tried from different accounts, changed the code, eve I just reinstall my Windows a few days ago and still having the same problem.

**Comment by mclmza** [comment]
Can be any write permission problem on your system?

**Comment by williamfgm96** [comment]
I don't think so
But just in case how can I check that?

**Comment by mclmza** [comment]
Which OS are you using?

**Comment by realistic829** [comment]
Hey bro, my code is working but there's a problem, after downloading 2 courses it downloads only 2 or 3 videos of third course and then gives the error that connection has been closed forcibly by remote server. It happens after downloading 2 courses. I think the problem is in the code because it downloads 2 courses successfully and only cause problem in the third. please help. Thanks!

**Comment by mclmza** [comment]
Try to open the cmd with administrator privileges, right click then 'Run as
administrator'. After this, execute the script from that shell

2018-04-21 17:24 GMT+02:00 williamfgm96 <notifications@github.com>:

**Comment by williamfgm96** [comment]
@mclmza it didn't work
Is there any way to see the errors on python?
I want to see what's going on

**Comment by williamfgm96** [comment]
The code crash

**Comment by mclmza** [comment]
Can you report the error?

**Comment by realistic829** [comment]
@mclmza Any solution to my problem brother?

**Comment by pheemmmy** [comment]
python :   File "lld.py", line 40
At line:1 char:1
+ python lld.py
+ ~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (  File "lld.py", line 40:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
    print cookie_jar._cookies
                   ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(cookie_jar._cookies)?

**Comment by mclmza** [comment]
@pheemmmy this error is related to your python version. are you sure that you are using Python 2.7?

**Comment by pheemmmy** [comment]
Great...thank you. 
I will uninstall my python. 
I am using python 3.6

**Comment by pheemmmy** [comment]
With Python 2.7, "pip install -r requirements.txt" isn't running

PS C:\Linkedin\linkedin> pip install -r requirements.txt
pip : The term 'pip' is not recognized as the name of a cmdlet, function, script file, or operable program. Check 
the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ pip install -r requirements.txt
+ ~~~
    + CategoryInfo          : ObjectNotFound: (pip:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by williamfgm96** [comment]
@pheemmmy Try adding 'pip' to Path, like with Python.

**Comment by ledi8** [comment]
Please help

`{'.linkedin.com': {'/': {'lang': Cookie(version=0, name='lang', value='"v=2&lang=en-us"', port=None, port_specified=False, domain='.linkedin.com', domain_specified=True, domain_initial_dot=False, path='/', path_specified=True, secure=False, expires=None, discard=True, comment=None, comment_url=None, rest={}, rfc2109=True), 'bcookie': Cookie(version=0, name='bcookie', value='"v=2&47412ae4-2b4d-417e-8b5f-a596283ca25e"', port=None, port_specified=False, domain='.linkedin.com', domain_specified=True, domain_initial_dot=True, path='/', path_specified=True, secure=False, expires=1587637643, discard=False, comment=None, comment_url=None, rest={}, rfc2109=False), 'lidc': Cookie(version=0, name='lidc', value='"b=SGST06:g=18:u=1:i=1524523791:t=1524610191:s=AQFbINYQJSJFZjP7IJm0dTdRaAbxmsRv"', port=None, port_specified=False, domain='.linkedin.com', domain_specified=True, domain_initial_dot=True, path='/', path_specified=True, secure=False, expires=1524610191, discard=False, comment=None, comment_url=None, rest={}, rfc2109=False)}}, '.www.linkedin.com': {'/': {'bscookie': Cookie(version=0, name='bscookie', value='"v=1&20180423224951d528c106-daf2-4b67-82ba-2179a6e67360AQH4GW_E7fVYPkwbyTPOxFl-HkXNK36M"', port=None, port_specified=False, domain='.www.linkedin.com', domain_specified=True, domain_initial_dot=True, path='/', path_specified=True, secure=True, expires=1587637643, discard=False, comment=None, comment_url=None, rest={'HttpOnly': None}, rfc2109=False), 'chp_token': Cookie(version=0, name='chp_token', value='AQHQ86HC9uHm1wAAAWL0svmEwRcD4FEHXm46ySMrRkC-XMCCO2I3iaWkZSSoObgT-DRE8ua0AbLrPrrKIL_wyAQTHw', port=None, port_specified=False, domain='.www.linkedin.com', domain_specified=True, domain_initial_dot=True, path='/', path_specified=True, secure=True, expires=None, discard=True, comment=None, comment_url=None, rest={}, rfc2109=False), 'JSESSIONID': Cookie(version=0, name='JSESSIONID', value='ajax:1665032769991291503', port=None, port_specified=False, domain='.www.linkedin.com', domain_specified=True, domain_initial_dot=True, path='/', path_specified=True, secure=False, expires=None, discard=True, comment=None, comment_url=None, rest={}, rfc2109=False)}}, 'www.linkedin.com': {'/': {'visit': Cookie(version=0, name='visit', value='"v=1&G"', port=None, port_specified=False, domain='www.linkedin.com', domain_specified=False, domain_initial_dot=False, path='/', path_specified=True, secure=False, expires=1587595785, discard=False, comment=None, comment_url=None, rest={}, rfc2109=True), 'leo_auth_token': Cookie(version=0, name='leo_auth_token', value='"GST:Zb3SXeAQWewoxiH1Zl8t5xLZF7G_cnPnlSUDUpzQWewgN1YpTv8sup:1524523792:fd616f4e8e343f65b1c424a993cbe517996e1f32"', port=None, port_specified=False, domain='www.linkedin.com', domain_specified=False, domain_initial_dot=False, path='/', path_specified=True, secure=False, expires=1524525585, discard=False, comment=None, comment_url=None, rest={}, rfc2109=True)}}}`
